core/src/main/java/org/apache/carbondata/core/cache/Cacheable.java

```diff
@@ -40,7 +40,7 @@
 
   /**
    * Method to be used for invalidating the cacheable object. API to be invoked at the time of
-   * removing the cacheable object from memory. Example at the of removing the cachebale object
+   * removing the cacheable object from memory. Example at removing the cacheable object
```

Comment:
Remove return tag in this interface

Reply:
I will do it in new PR. This PR focus on the typo.